packages/lesmis/src/views/DashboardView.js

```diff
@@ -178,6 +178,14 @@ const VerticalDivider = styled(MuiDivider)`
   height: 7rem;
 `;
 
+const PhotoOrMap = ({ vitals }) => {
+  if (vitals.isLoading) return null;
```

Comment:
Maybe better to place the loader here instead of the mini-map (or as well as in the mini-map)

Reply:
Yep that's actually where I put it

Reply:
(when I had it)